doc/plugins/org.polarsys.capella.ui.doc/html/04. User Interface/4.13. Activity Explorer.mediawiki

```diff
@@ -0,0 +1,16 @@
+= Activity Explorer =
+<br><br>
+{| border="1"
+|-
+| 
+[[Image:images/Information.png|48x48px]]
+|
+The Activity Explorer is the main interaction point with a Capella model.
+|}<br>
+After selecting one of the Arcadia perspective, it allows to:
+<ul>
+<li>create new diagrams depending on the type of activity the user want to address</li>
+<li>and open the already existing diagrams</li>
+</ul>
+[[Image:images/ActivityExplorer.png|1000x500px]]<br><br>

```

Comment:
The image doesnt seems to work when we open on github
https://github.com/eclipse/capella/pull/2224/files#diff-ed45a4a8838ab827ad4f3daba7f9d997dfaa9e6c30ff634f95e911baa00c311aR37

Mainly we have an online version where we links towards to mediawiki files.
https://github.com/eclipse/capella/wiki/User-manual#online-version

Reply:
It seems to be covered in Capella Overview / Activity Browser.

![image](https://user-images.githubusercontent.com/60220371/128978047-c66cd27a-a913-4a8c-955a-ea88b4d70d69.png)